repos/runtime.common.props

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
+   
+  <!--- Portable / TargetRid Overrides-->
+  <PropertyGroup>
+      <!-- OSX core-setup build always uses the portable RID, so override it -->
```

Comment:
(Nit) Formatting, too many spaces (through file)

This line also mentions core-setup.

Reply:
Fixed